lib/screens/play/current_card/card_actions/card_action.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+import 'package:treasure_of_the_high_seas/model/card/action/card_action.dart' as Model;
+
+import './card_action_cost.dart';
+import './card_action_reward.dart';
+import './card_action_text.dart';
+
+class CardAction extends StatelessWidget {
```

Comment:
Could we avoid naming this the same thing as the underlying model class? Thinking it would be good to have a naming convention for our components, so it's super clear from the filename which things pertain to UI and which don't.

So perhaps something like `CardActionPanel`?